e2e/e2e_test.go

```diff
@@ -47,7 +47,7 @@ var _ = Describe("e2e", func() {
 			// find a worker node
 			workers := &v1.NodeList{}
 			selector := labels.NewSelector()
-			req, _ := labels.NewRequirement("node-role.kubernetes.io/worker", selection.Exists, []string{})
+			req, _ := labels.NewRequirement("node-role.kubernetes.io/control-plane", selection.DoesNotExist, []string{})
```

Comment:
What do you mean by "kind does not use worker role labels"? 
In the last commit we use worker label in `hack/kind-config.yaml`

Reply:
kind does not add the label to the worker nodes, no matter you see in the config 🤷🏼‍♂️ 
Beside that, this matches the NHC default config.